wiki/Rules/en.md

```diff
@@ -57,17 +57,17 @@ If you do find yourself on the wrong end of our community rules, bad things will
 
 These bad things may include (*but are not limited to*):
 
-* The removal of your ability to chat in-game for a period of time (known as a silence)
-* The removal of any infringing content from your userpage, profile or uploaded beatmaps
-* The loss of the privilege to edit or alter your userpage or profile details
-* The loss of the privilege to play or qualify in officially hosted tournaments
+* The removal of your ability to chat in-game for a period of time (known as a silence).
+* The removal of any infringing content from your userpage, profile or uploaded beatmaps.
+* The loss of the privilege to edit or alter your userpage or profile details.
+* The loss of the privilege to play or qualify in officially hosted tournaments.
 
 Silences and other infringements **automatically** increase in duration near-exponentially with each infringement. Moderators do not control the length of time you are silenced for an offense - repeat offenders will find themselves locked out of the game for increasingly longer periods of time.
-    
+
 Certain offenses carry a minimum duration (usually 48 hours or more) if they are particularly severe, though they are almost exclusively reserved for severe infractions, such as sharing graphic content or unrepentant harassment of other users.
 
 If after several infringements you continue to break the rules, your account may be permanently and irrevocably restricted.
 
-Restricted accounts can not play online, set scores, chat with other users and are for all intents and purposes 'banned' from interaction with the community. Restriction for community rule violation can only be removed upon appeal to the game staff after 3 months have passed.
+Restricted accounts can not play online, set scores, chat with other users and are for all intents and purposes 'banned' from interaction with the community. Restriction for community rule violation can only be removed upon appeal to the game staff after 3 months have passed ([conditions may apply](/wiki/Help_Centre/Account_Restrictions#common-restriction-reasons-and-cooldowns)).
```

Comment:
```suggestion
Restricted accounts cannot play online, set scores, chat with other users and are for all intents and purposes "banned" from interaction with the community. Restriction for community rule violation can only be removed upon appeal to the support staff after 3 months have passed ([more conditions may apply](/wiki/Help_Centre/Account_Restrictions#common-restriction-reasons-and-cooldowns)).
```

Reply:
applied this in my commit along with other corrections

Reply:
missed some parts of it 👀 try using the "commit suggestion" button if u want to apply it directly next time, that way errors don't happen and I don't have to check again 👍